docs/source/examples/simulate_RFB_example.ipynb

```diff
@@ -0,0 +1,1151 @@
6J9AMMJ317tjRoVPJgibe33aimkJ369YtGjp0KL311lvUvHlzGjZsGN1yWvZQZVH9gggNlYa/27fbfjy3fffpY7cIMTHSSzi9tHcZYRpwm04J3cyJi5Oy3NuqET19Kg2d792zUwAWBAKw9Bz2OLPPncuOb9XKbhGkuVt3d9s1AZ6E+YMPHBBAUkzt9QeUxbVzwgR2kqpVbTMRmiaCc2AK/tkzKServVlQuAKwdKndYjDHPXvub14lwKGbkuXy5sZiexWRmjXZOeyuZ5qYKA1cbVXMli2TxS2Fz1pmy2Z/JDufyZg7104hEhOl0HtbHPiIpNGJg2mu+Gyvl5csxj6HUf397QCqKHT//vsveXh4UO3atWnUqFE0cuRIql27Nnl6etLOnTvVEElWVL8guLNN/fr22eJ37WLHFyhgt/MWH8wHBtpXrsVoZEnKAZY2zS64D2GBAvZZVvjb084RaHy8NOVsb37e//6THnZ2GYdMnZ3tcb46fpwd6+Njd92dgwelKWd7sxVx187q1e07nh4+lOav7YnM445bqarS2ALXJytXtt8nks8Y213I5MAB+5V7o5GoUCGHFVuuT/bqZfcp6LPPHPQf42UIcubMPFdKasLCJHOznSbCxEQpXs2R3IJ8JsNu/1LeD7ly2d4Ply5Jlms7g8dM+0GmODyHUf397QCqKHTVqlWj8ePHp1k/fvx4qm73E1s7qHpB3L8vRTLam1Y+Lo7V6gOsi6FPRUSE5GZiqRiDtfTrx84xbJgdB8fGSi8fe31+ZsxwaAT644/s8Hz57A+WNRolI6NdUX08ZNjT0z4zQFKSlJ7fjuvJaJRmZgYOtL15Di+lCWSex9AiXJt64w37Bjk8n0KRInYdHxbmmD7J4dYtu/VrbtKx98cYNMj+wQFJWWDs0SdN4YOEPHnsSDxrNDKtGmAJMu2BJ8u001S6apWkR2WUEjMzLl92QKcyGomqVWMnmDrV9sZNg8dM64LbwK+/Snq1I/0gJ0KhsxFPT0+6YcF34/r16+Tp6amCRPKi6gXBQ6YaNnTMU5Y7udjhJ8KTCJcr51iWb26KL1HCjn+F+84VKmT/9JADI9C4OCl/7/ff29c8h2cIsOsdzPNjOGBZSklvYUdY4Y4dUhfev2+/CERMFwNsi5AlIuZ85ufn0IuHXr6UHJZMC9dbCc/sX7euY7elaarIzNJKpoH7Mbi4WBftbQke4VO4sM3/iNEo5VZ0tHpWQoL0k1qqTpUhvFJK9uz2J/XjD7n27W0+NC5O0oO++ca+5jkODfj4TEy2bPYHwA0bxs6RWT5HC8TFSQFCMudwdwih0NlI4cKF6Q8LTsnr1q2jIkWKqCCRvKh2QYSGSqkFHE0MzOs31qxp02EvXkhZIVavdkyE6Gjp37ElOpSSkpg2aY/vnCmmT0sbzSq8+woWdMjdiIgceIdeuCC9wB3xT+Ulj+zQrBs3ZofKECycMt347rs2HsjznpQt61j+l3fescuqEx7uuNHclI8/ZucaM8bGAz/6iB1ojwc+59UrydR49qxNh3I9Kls2eUrj8tiEL76w8UAerevIRcmVY29vm3NscMt9wYIOx9gQkTTgs3nM1rKlA1Mgyezcyc6RP7/N9xb3iMmfX/ZiNA4hFDobmTJlCuXMmZO++eYbOnjwIB06dIhmzJhBOXPmpK/sNYFrCNUuCO5YIkddHDvnuHiRhcBAedI08eBMm/QynlvKz8/xdPLctNK/v9WHmOqTs2Y51jyRuXHIpvQl/AVuTUmGjDDVrK9csfowHkzp7i5P+bJTp9j5cuSwIX1JfLwU0OBQJAFJCYlr17bpMH5b1qkjT05eHpRoU53fe/ekEFlHaxR36MDOM2WKTYfJqdwTSeXQbPLSOXuWHeTqSuRIiiyjkSVZttE0lpAg+dU6nJIpGe4NUKyYDdeX6WDPkUiE2FjJPefkSasPi4+XxsoyZMGRFaHQ2YjRaKS5c+dSoUKFUhILFypUiObNm0dGGcvrqIUqF0RUlBRCZ++0Umps9BNJSJCmEmxJfZcRPAVYs2Y2HMSdtiz4adoMNysULGj105JPFcuhT3LatmXntDq44u5d+V7gRJLvlQ0aKi+J60CwtBmm0427dll5EHdWKlDAcVNpaKg0yLHSxBQRIWVKkeu2jIhgASaADTOnvI6xTTdSOnB3hlq1rD5EbuWeiMW58PQlVo85u3dnB3Tr5rgA3L3F2io8JMUg5Mkjn1XqxQvJAnz5spUH9ejhuLWWw91zvvzS6kP4bZkvn7pJhC0hFDoHiIqKoihbk5RpHFUuCK75lC4tXxVzXubGyuRA/GHl7y/PVAIRi4oEbKgDzd8cHh52es+n4tUr5msDsGRqmWA0SnqwnFFbixfbaHzlL/C33pJHAG56bdjQqt15dK7BYON0eSZwd75Ro6zY2WiUEqbaG2acmlq1bHLk43E1FSrId1sSETVpYoN1IzJSsqLIUUPbNMrTynuMz1bLpdxz+L1m1c/x4IE0yLHiXs4UPtgLCLBqsCdHLEZ6cFdZq4qZhIdLxbVtsKqlCw9Br1bNqt2TkqSMMVryneMIhU5ghuIXREKCZL9eskS+8/LpiezZM7VuOBowlRFlyrDzWlXDslMn+d8cHTuyc1qRp4mnfPPyYs9Nubh3T5ohybQGakyM5DFuTx5CS9y9a4MAUlUIGUteEpFUjaxMGSt2PnRI8nOSq6I9r1fVoUOmu758KQUIOxLtbYlZs9h5rUpXsXCh5EMo1wwI16R++inTXXlskcEgW9WwFLhfpVXXGd/ZykFJpphON1qRDWDrVsllQK7LkbNgATt3kyZW7DxtGtu5bl15Gn/8WMrZacX0LfeI8fW1PVOKEgiFzgqqV69Oz5Kv4mrVqlH16tXTXbI6il8Qa9dKpjE57demeacy8RPh6a28vR2uGJYGbmzq2TOTHR88kOaiZCqkTkTSFJMV1yYvlypLvc1U8NF9psEm3NerZEl5zUJcgJUrM9wtLEwyANhaFSI
```

Comment:
I would remove the `=` after the parameter names, it's sufficient to say that those are parameters.